views/page/reporting/index.html

```diff
@@ -28,8 +28,8 @@ <h2 class="govuk-heading-m">Booking information</h2>
                     <li class="course list-details u--border--bgrey">
                         <form action="/reporting/booking-information" method="post">
                             {{ govukDateInput({
-                                id: "start-date",
-                                name: "start-date",
+                                id: "booking-start-date",
+                                name: "booking-start-date",
```

Comment:
Could we please follow the same naming convention for id and name without a hyphen or with a hyphen as in other places in this file. Also can we please keep id and name in the same order if possible.